src/main/java/uk/gov/hmcts/dts/fact/services/validation/ValidationService.java

```diff
@@ -27,4 +29,13 @@ public ValidationService(PostcodeValidator mapitPostcodeValidator) {
             .filter(postcode -> !mapitPostcodeValidator.postcodeDataExists(postcode.replaceAll("\\s+","")))
             .collect(toList());
     }
+
+    /**
+     * Validates that the given local authority name exists as a local authority.
+     * @param localAuthorityName The local authority name
+     * @return A boolean indicating if the local authority name is valid or not
+     */
+    public boolean validateLocalAuthority(String localAuthorityName) {
```

Comment:
Very constant, can add final keyword to the localAuthorityName if it doesn't change

Reply:
Fixed.